Added section on environment variables

Rajeev-K opened the pull request:
Added a section on environment variables. Also removed the word "Workbench" from the first sentence since model data collection is not a Workbench feature.

azurecla commented:
Hi __@nickheppleston__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mimig1 commented:
Please accept.

Rajeev-K committed:
Added section on environment variables

Added a section on environment variables. Also removed the word "Workbench" from the first sentence since model data collection is not a Workbench feature.

garyericson commented on articles/machine-learning/preview/how-to-use-model-data-collection.md:
```diff
@@ -34,6 +34,12 @@ Then run the following command:
 
+## Set environment variables
+
+Model data collection depends on two environment variables. AML_MODEL_DC_STORAGE_ENABLED must be set to true (all lowercase) and AML_MODEL_DC_STORAGE must be set to the connection string for the Azure Storage account where you want to store the data.
```
Would be helpful if you put quotes around "true" to indicate it's a literal value.

garyericson commented on articles/machine-learning/preview/how-to-use-model-data-collection.md:
```diff
@@ -34,6 +34,12 @@ Then run the following command:
+
+Model data collection depends on two environment variables. AML_MODEL_DC_STORAGE_ENABLED must be set to true (all lowercase) and AML_MODEL_DC_STORAGE must be set to the connection string for the Azure Storage account where you want to store the data.
+
+These environment are already set for you when the Web service is running on a cluster in Azure. When running locally you have to set them yourself. If you are using Docker, use the -e parameter of docker run command to pass environment variables.
```
1st sentence: Add "variables" - "These environment variables..."
1st sentence: "Web" should be lowercase - "...when the web service is running..."
3rd sentence: Add "the" - "...parameter of the docker run command..."
